src/plays/git-hub-profile-search/Readme.md

```diff
@@ -0,0 +1,27 @@
+# GitHub Profile Search
```

Comment:
Please provide more context of the play.

Reply:
added